saleor/plugins/webhook/tests/test_tax_webhook_tasks.py

```diff
@@ -0,0 +1,103 @@
+from unittest import mock
+
+import pytest
+
+from ....webhook.event_types import WebhookEventType
+from ....webhook.models import Webhook, WebhookEvent
+from ..tasks import trigger_tax_webhook_sync
+from ..utils import parse_tax_data
+
+
+@pytest.fixture
+def tax_checkout_webhooks(tax_app):
+    webhooks = [
+        Webhook(
+            name=f"Tax checkout webhook no {i}",
+            app=tax_app,
+            target_url=f"https://www.example.com/tax-checkout-{i}",
+        )
+        for i in range(3)
+    ]
+    Webhook.objects.bulk_create(webhooks)
+    WebhookEvent.objects.bulk_create(
+        WebhookEvent(
+            event_type=WebhookEventType.CHECKOUT_CALCULATE_TAXES,
+            webhook=webhook,
+        )
+        for webhook in webhooks
+    )
+
+    return webhooks
+
+
+@mock.patch("saleor.plugins.webhook.tasks.send_webhook_request_sync")
+def test_trigger_tax_webhook_sync(
+    mock_request,
+    tax_checkout_webhook,
+    tax_data_response,
+):
+    # given
+    mock_request.return_value = tax_data_response
+    event_type = WebhookEventType.CHECKOUT_CALCULATE_TAXES
+    data = '{"key": "value"}'
+
+    # when
+    tax_data = trigger_tax_webhook_sync(event_type, data)
+
+    # then
+    mock_request.assert_called_once_with(
+        tax_checkout_webhook.pk,
+        tax_checkout_webhook.target_url,
+        tax_checkout_webhook.secret_key,
+        event_type,
+        data,
+    )
+    assert tax_data == parse_tax_data(tax_data_response)
+
+
+@mock.patch("saleor.plugins.webhook.tasks.send_webhook_request_sync")
+@pytest.mark.parametrize("no_of_unsuccessful_webhooks", [0, 1, 2])
+def test_trigger_tax_webhook_sync_multiple_webhooks(
+    mock_request,
+    tax_checkout_webhooks,
+    tax_data_response,
+    no_of_unsuccessful_webhooks,
+):
+    # given
+    mock_request.side_effect = (
+        tax_data_response if i == no_of_unsuccessful_webhooks else {}
```

Comment:
I am not convinced that `no_of_unsuccessful_webhooks` is really needed. Can't we do:
```
side_effect = [{}, {}, tax_response] 
```
This should be clear enough that this will iterate over a webhooks. 
You could add separate test (maybe it is already added), where your side_effect looks like [tax_response, somethin1, something2] and assert on mock confirms that we hit only one request (we fetched first object)